test/corpus/sections.txt

```diff
@@ -424,3 +424,121 @@ Disassembly of section __libc_freeres_fn:
         (byte)
         (byte))
       (instruction))))
+
+================================================================================
+Disassembly Section With Raw Data
+================================================================================
+
+Disassembly of section .data:
+
+20000000 <SystemCoreClock>:
+20000000:	00f42400                                .$..
+
+20000004 <uwTickPrio>:
+20000004:	00000010                                ....
+
+20000008 <uwTickFreq>:
+20000008:	00000001                                ....
+
+2000000c <__sglue>:
+2000000c:	00000000 00000003 20000200              ........... 
+
+20000018 <_impure_data>:
+20000018:	00000000 20000200 20000268 200002d0     ....... h.. ... 
+	...
```

Comment:
Do byte dumps actually continue to the next line like this or is that a typo?

Reply:
Something different, it's like filler zeroes